botbuilder-instumentation/DependencyResolver.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using BotBuilder.Instrumentation.Managers;
+using System.Configuration;
+using BotBuilder.Instrumentation.Instumentation;
+
+namespace BotBuilder.Instrumentation
+{
+    public class DependencyResolver
+    {
+        private static DependencyResolver _dependencyResolver;
+
+        private DependencyResolver() { }
+
+        public static DependencyResolver Current => _dependencyResolver ?? (_dependencyResolver = new DependencyResolver());
+
+        public InstrumentationSettings CreateBasicSettings()
+        {
+            return new InstrumentationSettings
+            {
+                InstrumentationKeys = new List<string>(new[] { ConfigurationManager.AppSettings["InstrumentationKey"] }),
+                OmitUsernameFromTelemetry =
+                    Convert.ToBoolean(ConfigurationManager.AppSettings["InstrumentationShouldOmitUsernameFromTelemetry"])
+            };
+        }
+
+        public SentimentManager CreateDefaultSentimentManager()
+        {
+            return new SentimentManager(
+                ConfigurationManager.AppSettings["TextAnalyticsApiKey"],
+                ConfigurationManager.AppSettings["TextAnalyticsMinLength"],
+                ConfigurationManager.AppSettings["CognitiveServiceApiEndpoint"]
+            );
+        }
+
+        public InstrumentationSettings CreateSettingsWithCognitiveServices()
+        {
+            var settings = CreateBasicSettings();
+            settings.SentimentManager = CreateDefaultSentimentManager();
+            return settings;
+        }
+
+        public BotFrameworkApplicationInsightsInstrumentation CreateBasicInstrumentation()
```

Comment:
Should return `IBotFrameworkApplicationInsightsInstrumentation`
In addition, this class should help the user to bootstrap instantly. By creating an instance we are asking the user to persist the instance himself. But we can do it for them in this class. 
I suggest to make this methods private, and expose 2 properties, something like:
```
public IBotFrameworkApplicationInsightsInstrumentation DefaultInstrumentationWithCognitiveServices ;
public IBotFrameworkApplicationInsightsInstrumentation DefaultBasicInstrumentation;
```
And they both act as a singleton - they create an instance and we take care of persisting it, and making the life of the user as simple as possible.

It means this file is optional and the user can choose to instantiate the system differently but we provide an out-of-the-box functionality.
I hope it makes sense.